### PR TITLE
added definition of technical debt

### PR DESCRIPTION
**Pull request by shelleymcq**
## This PR fixes...

This adds the definition of 'Technical Debt' to the dictionary.

## What I did...

I updated the technical-debt-1.md file to include the definition of 'technical debt' and a source url.

## How to test...

Follow this link for the netlify preview: https://deploy-preview-218--cherryontech.netlify.app/
Use the top navbar to navigate to 'Dictionary'.
Enter 'technical debt' in the search.

## I learned...

I learned how to use `git switch` instead of `checkout` when I created my new branch.

### CODE REVIEW

**Comment by cherryOnTechAdmin** [comment]
Hi Shelly!! It looks like there is something breaking in the build. You can see the error here: https://app.netlify.com/sites/cherryontech/deploys/650333befca106000844fdd0#L88 

I believe you are missing a closing single quote at the end of your definition, causing the parsing to break. Try pushing the fix and see if the build passes! It will automatically retrigger the build when you push your commit.

**Comment by alliequintano** [comment]
Hi Shelly!! It looks like there is something breaking in the build. You can see the checks below have failed and further inspect the error here: https://app.netlify.com/sites/cherryontech/deploys/650333befca106000844fdd0#L88

I believe you are missing a closing single quote at the end of your definition, causing the parsing to break. Try pushing the fix and see if the build passes! It will automatically retrigger the build when you push your commit.

**Comment by shelleymcq** [comment]
Thanks Allie! The closing quote still failed, so I tried removing all quotation marks, but that didn't work either. 

I haven't been able to get any of the gridsome commands to work locally. Could that have something to do with it?

**Review by KatKowalik — APPROVED** [review]
Looks good!

File: src/assets/content/dictionary/technical-debt-1.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Technical Debt
-definition:
-sources: []
+definition: Technical debt, also known as code debt, refers to the cost or consequences of prioritizing speed or ease of implementation over quality. In software development, this can mean making coding or design decisions that will need to be addressed in the future in order to get a viable solution into production sooner. While often necessary in the short term, this debt can add up and become more problematic or expensive over time.
+- sourceurl: https://enterprisersproject.com/article/2020/6/technical-debt-explained-plain-english
```
Comment (alliequintano) [inline src/assets/content/dictionary/technical-debt-1.md]: ah i see! Take a look at the example again: https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md#tech-dictionary-example

You see that you need to keep the line 
```- sources:```
and then the `sourceurl` is a key within the sources array.

**Comment by TanyaPina** [comment]
Looks great Shelley!
Not blocking for this PR: a thing to note is that there seems to be two instances of "technical debt" that exists on the website. If this is an error I can either set it as an issue to be addressed later or make a PR to fix it. If neither are necessary that is ok too @alliequintano

**Review by TanyaPina — APPROVED** [review]

**Comment by shelleymcq** [comment]
Thanks, Tanya! I was wondering about that too. I could just delete the .md file for the second one with this PR if Allie is cool with that.

**Comment by alliequintano** [comment]
> I could just delete the .md file for the second one with this PR if Allie is cool with that.

@shelleymcq heck yes! nice job reviewing @TanyaPina @KatKowalik!

**Comment by avinashi10** [comment]
Hey Shelley - the definition looks great! I would recommend slightly editing your Pull Request 'How to test...' section. If the tester goes to the live site they won't see the definition for technical debt as your branch hasn't been merged yet. Instead instruct them on how to run your branch locally or use the Netlify deploy preview above to test your changes.

**Review by YashsviG — APPROVED** [review]

**Pull request merged**